Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+lxml = "*"
+requests = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
I understand the desire for using the latest and greatest, but if we aim 3.6+ compatibility, we basically support the most popular OS: Ubuntu LTS.